packages/charts/src/charts/pie/pie-chart.component.tsx

```diff
@@ -130,15 +131,35 @@ export const PieChart: FC<Props> = ({
             <Tooltip mode={tooltipSettings.mode} hasArrow={false}>
               {tooltipSelectors && (
                 <BulletList
-                  list={getCircularChartTooltipContent({
+                  items={getCircularChartTooltipContent({
                     data,
                     keys,
                     labelSelector,
                     selectors: tooltipSelectors,
                     disabledLabels,
                     formatValue: formatTooltip,
                   })}
-                  typography={tooltipSettings.labels.typography}
+                  renderItem={(_idx, item) =>
+                    typeof item.data === 'string' ? (
+                      <Text {...tooltipSettings.labels.typography}>
+                        {item.data}
+                      </Text>
+                    ) : (
+                      <TooltipContent>
+                        <Text {...tooltipSettings.labels.typography}>
+                          {item.data.label}
+                        </Text>
+                        <Text {...tooltipSettings.values.typography}>
+                          {item.data.value}
+                        </Text>
+                        {item.data.change && (
+                          <Text {...tooltipSettings.labels.typography}>
+                            {item.data.change}
+                          </Text>
+                        )}
+                      </TooltipContent>
```

Comment:
Maybe it would be better to export render function from renderItem to separate component in order to avoid code duplication in pie-char.component and donut-chart.component ?